make/test.mk

```diff
@@ -196,7 +196,8 @@ create-host-resources:
 ifeq ($(HOST_REPO_PATH),)
 	$(eval HOST_REPO_PATH = /tmp/codeready-toolchain/host-operator)
 endif
-	oc apply -f deploy/host-operator/${ENVIRONMENT}/ -n ${HOST_NS}
+	# ignore if these resources already exist (nstemplatetiers may have already been created by operator)
+	-oc create -f deploy/host-operator/${ENVIRONMENT}/ -n ${HOST_NS}
```

Comment:
In some test executions, I had troubles with the NSTemplateTiers whose TierTemplate version was still `-000000` and I suspect that the `oc apply` command was executed after the operator already started, which means that the NSTemplateTier resources were reset again, and the e2e tests would fail. So I thought we may want to skip the `oc apply` (ie, ignore if `oc create` fails) when there's such a race condition

Reply:
in which cases did it happen? when you executed either `make test-e2e` or `make dev-deploy-e2e`? I haven't seen such an error, that's why I'm asking...

Reply:
it happened when running `IMAGE_BUILDER=docker make test-e2e` locally